From: Takashi Iwai <tiwai@suse.de>
Subject: struct acpi_ec kABI workaround
Patch-mainline: Never, kABI workaround
References: git-fixes

The patch
  patches.suse/ACPI-EC-Install-address-space-handler-at-the-namespa.patch
dropped a field from struct acpi_ec, and it breaks kABI.
Revive the dropped field as a placeholder.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 drivers/acpi/internal.h |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/acpi/internal.h
+++ b/drivers/acpi/internal.h
@@ -176,6 +176,7 @@ enum acpi_ec_event_state {
 
 struct acpi_ec {
 	acpi_handle handle;
+	acpi_handle address_space_handler_holder; /* FIXME: placeholder for kABI compatibility */
 	int gpe;
 	int irq;
 	unsigned long command_addr;
